doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst

```diff
@@ -22,7 +20,7 @@ It consists of two main concepts: the SUIT envelope and the SUIT manifest.
 * The SUIT envelope acts as a secure container for transporting firmware updates, encapsulating the firmware binary and its manifest.
 * The SUIT manifest is a structured file with metadata essential for the update process, including firmware version, size, and hash for integrity verification.
 
-During the first build of the ``nrf54h_suit_sample``, default manifest templates are provided.
+During the first build of the :ref:`nrf54h_suit_sample`, default manifest templates are provided.
```

Comment:
Is the link **:ref:`nrf54h_suit_sample`** intends to link to the sample here and at all other instances?

Reply:
Yes, I believe so

Reply:
I believe this UG is specifically using that sample to demonstrate how to customize, however you do raise a good point in that other samples can perhaps be mentioned throughout the existing SUIT docs... at the very list, some kind of link to all available SUIT samples we have... will make a ticket for that, aimed at RC2.